config/environments/production.rb

```diff
@@ -1,5 +1,7 @@
+require 'active_support/core_ext/integer/time'
```

Comment:
Do we need this? I don't see any lines requiring `time` unlike in test.rb/development.rb.

Reply:
You're right, the only line that uses this is commented out. Removed